client/post-editor/media-modal/gallery/fields.jsx

```diff
@@ -144,13 +151,26 @@ export default React.createClass( {
 		);
 	},
 
+	renderReverseOrderButton() {
+		const { translate } = this.props;
+
+		return (
+			<EditorMediaModalFieldset>
+				<Button onClick={ this.updateReverseOrder } disabled={ this.props.settings.orderBy === 'rand' } >
```

Comment:
Do you need `updateReverseOrder` as a function on the instance, or could you just pass `this.props.onReverse` directly? This would also help avoid the need for any checking of truthiness on the prop, as `<Button />` is responsible for handling this.